roles/deploy/tasks/deploy.yml

```diff
@@ -64,6 +64,7 @@
     "
   args:
     chdir: "{{ build_path }}"
+    creates: "{{ build_path }}/.bundle/config"
```

Comment:
TIL about this interestingly named argument. I would have expected more generic "if" style code, which I thought ansible also supports.

All good, this is clearly the standard way to do it, it's just new to me.